doc/refman/RefMan-pro.tex

```diff
@@ -103,6 +103,14 @@ \section{Switching on/off the proof editing mode}
 This command is available in interactive editing proof mode to give up
 the current proof and declare the initial goal as an axiom.
 
+\variant {\tt Unset Keep Admitted Variables} \optindex{Keep Admitted Variables}
+
+By default in sections, an {\tt Admitted} lemma is generalized by all
```

Comment:
Wording nit: "generalized over all" instead of "generalized by add"

Reply:
@SkySkimmer You didn't address this comment :) but then I think I bothered you enough on this PR already...

Reply:
That's a github illusion, I removed that commit.

Reply:
Or maybe it didn't get removed for some reason? It's gone now.